web/locales/sv.json

```diff
@@ -17,7 +17,7 @@
     },
     "country-panel": {
         "source": "källa",
-        "carbonintensity": "Utsläpp av koldioxid",
+        "carbonintensity": "Utsläpp av CO2",
```

Comment:
Why the discrepancy between country-panel.carbonintensity and tooltip.carbonintensity?
Let's keep koldioxid for all?

Reply:
It is a hack to try to remove the new line in 
![image](https://user-images.githubusercontent.com/493140/33872939-55a2ec8e-df19-11e7-8b5b-32c50960279d.png)

But, it is probably beter to fix the layout issue.
I can restore it to "koldioxid"

Reply:
ok got it. We shouldn't change the translation in order to fit on one line (we should at the UI instead).
Furthermore, you won't have a guarantee that the hack works because on small devices the space might small enough to cause a linebreak.

Reply:
You're right, I've restored the string